### PR TITLE
Ensdf processing delta

### PR DESCRIPTION
**Pull request by howland**
Followup to #797 @rachelslaybaugh 

Adds delta, gtol, and three of the four hsicc ensdf evaluation tools to PyNE.

### CODE REVIEW

**Comment by scopatz** [comment]
@howland I supposse the error is expected here as well?

**Comment by howland** [comment]
@scopatz yes - The other pr was failing because of some new change in Python3, I'm still working on getting all the dependencies and everything needed for Python3-PyNE on my local system.  I should have this all resolved by tomorrow, if not I'll remove the two evaluations tools that are failing and have a separate followup pr (for those two) when i've fixed them.

**Comment by scopatz** [comment]
Sounds good.  BTW, I use conda for basically everything but the compiler at this point. If you are running into dep issues, it is a good way to test things out by creating a throw-away env.

**Comment by howland** [comment]
@scopatz This one is good to go for a review as well.  I'm working on setting up conda right now.

File: pyne/ensdf_processing.py [hunk 1]
```diff
@@ -74,6 +74,30 @@ def alphad(inputdict_unchecked):
...
     return inputdict_unchecked
 
+def delta(inputdict_unchecked):
+    """
+    This function calculates the best values of mixing ratios based of its analysis of
+    the angular correlation and conversion coefficient data.
+
+    Input Dictionary Required Key Pair Value:
+        input_file : input ensdf file
+        output_file : file for output to be written to (doesn't have to exist)
```
Comment (scopatz) [inline pyne/ensdf_processing.py]: Your docstrings should use the numpydoc format.

File: setup.py [hunk 2]
```diff
@@ -177,6 +177,12 @@ def download_decay():
     return True
 
 ALPHAD_H = os.path.join('build', 'src/alphad')
+DELTA_H = os.path.join('build', 'src/delta')
+GTOL_H = os.path.join('build', 'src/gtol')
+HSICC_H = os.path.join('build', 'src/hsicc')
+HSMRG_H = os.path.join('build', 'src/hsmrg')
+SEQHST_H = os.path.join('build', 'src/seqhst')
```
Comment (scopatz) [inline setup.py]: I'd be nice if you created a list or dict for these and then looped over it :)

File: src/ensdf_processing/HSICC/bldhst.for [hunk 3]
```diff
@@ -0,0 +1,131 @@
+C***********************************************************************01 00010
+C*                                                                      01 00020
```
Comment (scopatz) [inline src/ensdf_processing/HSICC/bldhst.for:2]: Excuse my ignorance, but is `*.for` a common extension.  Also, do we have permission to redistribute these?

Comment (howland) [inline src/ensdf_processing/HSICC/bldhst.for:2]: @rachelslaybaugh we think we do have permission, correct?
@scopatz common extension?  The extension (.f, .for, etc) varies, I left it in the form it was distributed with.

Comment (rachelslaybaugh) [inline src/ensdf_processing/HSICC/bldhst.for:2]: Yes, we have permission.

Comment (scopatz) [inline src/ensdf_processing/HSICC/bldhst.for:2]: Thanks @howland

File: pyne/ensdf_processing.py (hunk 1 above)
Comment (howland) [inline pyne/ensdf_processing.py]: Better?

**Comment by howland** [comment]
@scopatz I reformatted both the documentation and executable copying (its now done by iterating over a list) in both this PR and #811.  All your comments should be addressed.

File: setup.py (hunk 2 above)
Comment (howland) [inline setup.py]: Done

File: pyne/ensdf_processing.py (hunk 1 above)
Comment (howland) [inline pyne/ensdf_processing.py]: Reformatted!

File: pyne/ensdf_processing.py [hunk 4]
```diff
@@ -74,15 +74,49 @@ def alphad(inputdict_unchecked):
...
     return inputdict_unchecked
 
+def delta(inputdict_unchecked):
+    """
+    This function calculates the best values of mixing ratios based of its analysis of
+    the angular correlation and conversion coefficient data.
+
+    Parameters
+    ----------
+    Input_dictionary : dictionary
```
Comment (scopatz) [inline pyne/ensdf_processing.py]: `Input_dictionary` should be the actual name of the argument, ie `inputdict_unchecked`

File: pyne/ensdf_processing.py [hunk 5]
```diff
@@ -74,15 +74,49 @@ def alphad(inputdict_unchecked):
...
+        dictionary that must have the following key-pair values:
+            input_file : input ensdf file
+            output_file : file for output to be written to (doesn't have to exist)
+
+    Notes
+    -----
+    Output Dictionary Values:
```
Comment (scopatz) [inline pyne/ensdf_processing.py]: Give the name of the return variable here, or just `rtn`.  Also give the variable type after the colon.

File: pyne/ensdf_processing.py [hunk 6]
```diff
@@ -74,15 +74,49 @@ def alphad(inputdict_unchecked):
...
+    the angular correlation and conversion coefficient data.
+
+    Parameters
+    ----------
+    Input_dictionary : dictionary
+        dictionary that must have the following key-pair values:
+            input_file : input ensdf file
+            output_file : file for output to be written to (doesn't have to exist)
+
+    Notes
```
Comment (scopatz) [inline pyne/ensdf_processing.py]: Shouldn't this be `Returns`, rather than `Notes`

**Pull request merged**